cmd/argo/commands/stop.go

```diff
@@ -31,28 +46,75 @@ func NewStopCommand() *cobra.Command {
   argo stop @latest
```

Comment:
suggestion - add an example of the new usages

Reply:
Thanks! added examples for both `argo stop` and `argo terminate` in the latest commit:

![Screen Shot 2021-04-28 at 12 44 39 AM](https://user-images.githubusercontent.com/1311594/116347831-f587b780-a7ba-11eb-9907-f825e232b70e.png)

![Screen Shot 2021-04-28 at 12 45 06 AM](https://user-images.githubusercontent.com/1311594/116347845-fb7d9880-a7ba-11eb-9ab7-f3acb66aa0f8.png)